application/views/users/view.php

```diff
@@ -0,0 +1,33 @@
+<main role="main" class="flex-shrink-0">
+  <div class="container">
+      <h1>All informetion of Users</h1>
+      <table class="table table-striped table-hover">
+          <thead>
+              <tr>
+              <th scope="col">#</th>
+              <th scope="col">First Name</th>
+              <th scope="col">Last Name</th>
+			  <th scope="col">E-mail</th>
+			  <th scope="col">số điện thoại</th>
+              <th scope="col">Action</th>
+              </tr>
+          </thead>
+          <tbody>
+              <?php foreach ($users as $user): ?>
```

Comment:
This screen is to show information of only one user. So that, you have to select the user and show here. It's not a list of users.

Reply:
I misunderstood what the question required. I'll change how the view page work.